courses/models.py

```diff
@@ -0,0 +1,57 @@
+"""
+Course models
+"""
+import logging
+
+from django.db import models
+
+
+log = logging.getLogger(__name__)
+
+
+class Program(models.Model):
```

Comment:
Can we copy over `TimestampedModel` and use that here? Do you think that would be useful?